testing/bdd/features/segmentation/duplicate_sequence_id+nun.feature

```diff
@@ -0,0 +1,37 @@
+
+
+Feature: Discard document with already seen pair of sequence identifier and sequence number 
+
+    Examples:
+    | xml_file            |   
+    | sequence_id_num.xml |   
+    
+
+  # SPEC-CONFORMANCE: R107
+  Scenario: Discard document
+    Given an xml file <xml_file>
+    And it has sequence identifier <seq_id>
+    And it has sequence number <seq_n>
+    When a previous document exists with sequence identifier <prev_seq_id> and sequence number <prev_seq_n>
```

Comment:
I think it makes more sense here to say that when the document is made available and then another document with the same id and number becomes available then when trying to add the second document it is not added. This BDD case never creates the first document, so all the work has to be done in "When a previous document exists..." which presumably creates the first document and adds it?